keys.json

```diff
@@ -1,9 +1,11 @@
 {
+    "ace":           { "skip": true, "//": "API randomly fails and returns a HTML error" },
     "alpaca":        { "skipWs": true },
     "bkex":          { "skip": true },
     "binance":       { "skipWs": true, "httpProxy": "http://51.83.140.52:11230" },
     "binanceusdm":   { "skipWs": true, "httpProxy": "http://51.83.140.52:11230" },
     "binancecoinm":  { "skipWs": true, "httpProxy": "http://51.83.140.52:11230" },
+    "binanceus":     { "skipWs": true, "httpProxy": "http://51.83.140.52:11230" },
```

Comment:
@carlosmiei
Proxying binanceus through an european proxy seems wrong. If any, this should be using a different proxy, which is located in the US.

I assume you're seeing binance.us errors since #16825 (we see this as well with very recent ccxt versions.).

While this PR will fix your CI - it will NOT fix the underlying problem, which is that you can't initialize `binanceus()` anymore if you're on an US IP.


``` python
import ccxt
ct = ccxt.binanceus()
ct.load_markets()
```

```
Traceback (most recent call last):
  File "/workspaces/ccxt/python/ccxt/base/exchange.py", line 625, in fetch
    response.raise_for_status()
  File "/home/codespace/.local/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 451 Client Error:  for url: https://eapi.binance.com/eapi/v1/exchangeInfo

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/workspaces/ccxt/python/ccxt/base/exchange.py", line 1460, in load_markets
    markets = self.fetch_markets(params)
  File "/workspaces/ccxt/python/ccxt/binance.py", line 1697, in fetch_markets
    promises.append(self.eapiPublicGetExchangeInfo(params))
  File "/workspaces/ccxt/python/ccxt/base/exchange.py", line 505, in inner
    return entry(_self, **inner_kwargs)
  File "/workspaces/ccxt/python/ccxt/binance.py", line 6265, in request
    response = self.fetch2(path, api, method, params, headers, body, config, context)
  File "/workspaces/ccxt/python/ccxt/base/exchange.py", line 2830, in fetch2
    return self.fetch(request['url'], request['method'], request['headers'], request['body'])
  File "/workspaces/ccxt/python/ccxt/base/exchange.py", line 643, in fetch
    self.handle_http_status_code(http_status_code, http_status_text, url, method, http_response)
  File "/workspaces/ccxt/python/ccxt/base/exchange.py", line 1753, in handle_http_status_code
    raise ErrorClass(self.id + ' ' + method + ' ' + url + ' ' + codeAsString + ' ' + reason + ' ' + body)
ccxt.base.errors.ExchangeNotAvailable: binanceus GET https://eapi.binance.com/eapi/v1/exchangeInfo 451  {
  "code": 0,
  "msg": "Service unavailable from a restricted location according to 'b. Eligibility' in https://www.binance.com/en/terms. Please contact customer service if you believe you received this message in error."
```

It's most likely caused by this segment:
https://github.com/ccxt/ccxt/blob/55162d8245e2e87aee46e58610e3dc1e5f0a7dc4/js/binance.js#LL1699C7-L1702C10

Reply:
@xmatthias Oh ok, makes total sense, I will try to find an alternative solution and remove the proxy. Thanks for letting us know